front-office/breadcrumbs.md

```diff
@@ -6,3 +6,4 @@
 - Previous levels will be underline on hover to show that they are clickable
 
 ## POSITION AND PLACEMENT RULES
+- Keep the same position
```

Comment:
Here we should write exactly what is the rule for positioning.

It's like a design system's guidelines.
"Breadcrumbs should be located on pages at x leve of depth in the navigation
If the page is reached after a search, the breadcrumbs should offer the option to go back to the search results, etc.
On the top left part of the screen" etc.